website2/src/components/homePage/bash/index.tsx

```diff
@@ -0,0 +1,54 @@
+import { Text } from "../../text";
+import { SampleCode } from "../../sampleCode";
+import { Terminal } from "../../terminal";
+import styles from "./styles.module.css";
+import { Command, TypingText } from "../../typingText";
+
+const CODE = `export const backend = garn.go.mkGoProject({
+  description: "Go backend",
+  src: ".",
+  goVersion: "1.20",
+})
+  .withDevTools([pkgs.protobuf, pkgs.protoc_gen_go])
+  .addExecutable("codegen")\`protoc --go_out=out protobufs/*.proto\`;`;
```

Comment:
As mentioned in the previous PR this was changed in a previous version of garn.

```suggestion
  .addExecutable("codegen", "protoc --go_out=out protobufs/*.proto");`;
```